app/v1/models/officesmodel.py

```diff
@@ -0,0 +1,35 @@
+politicaloffices_list = []
+
+class OfficeModel:
+    
+    
+    office_id= 1
```

Comment:
You don't need this variable since you are getting the `office_id` from the `length` of list holding all offices on line 13